content/docs/tasks/policy-enforcement/control-headers/index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Control Headers and Routing
+description: Shows how to modify headers using policy adapters.
+weight: 20
+keywords: [policies,routing]
+---
+
+This task demonstrates how to use a policy adapter to manipulate request headers and routing.
+
+## Before you begin
+
+* Set up Istio on Kubernetes by following the instructions in the
+  [Installation guide](/docs/setup/kubernetes/).
+
+    {{< warning >}}
+    Policy enforcement **must** be enabled in your cluster for this task. Follow the steps in
+    [Enabling Policy Enforcement](/docs/tasks/policy-enforcement/enabling-policy/) to ensure that policy enforcement is enabled.
+    {{< /warning >}}
+
+* Follow the set-up instructions in the [ingress task](/docs/tasks/traffic-management/ingress/) to configure an ingress using a gateway.
+
+* Customize the [virtual service](/docs/reference/config/networking/v1alpha3/virtual-service/)
+  configuration for the `httpbin` service containing two route rules that allow traffic for paths `/headers` and
+  `/status`:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: VirtualService
+    metadata:
+      name: httpbin
+    spec:
+      hosts:
+      - "*"
+      gateways:
+      - httpbin-gateway
+      http:
+      - match:
+        - uri:
+            prefix: /headers
+        - uri:
+            prefix: /status
+        route:
+        - destination:
+            port:
+              number: 8000
+            host: httpbin
+    EOF
+    {{< /text >}}
+
+## Output-producing check adapters
```

Comment:
remove the word "check", we don't use that with customers.

Reply:
Done